### PR TITLE
Apply fixes for JBUG upto Beta6

### LINKED ISSUE #765: Update JBUGs for slbeta6
Update all JBUG comments to remove any that are fixed for beta6 and add bug numbers for other.

### PR DESCRIPTION
**Pull request by gimantha**
Partially fixes #765

### CODE REVIEW

File: compiler/main.bal [hunk 1]
```diff
@@ -64,10 +64,8 @@ public function main(string[] filenames, *Options opts) returns error? {
                 errorFileCount += 1;
                 dPrinter.print(err.detail());
             }
-            // JBUG: #34014
-            // can't use else { check err; }
-            else if err != () {
-                return err;
+            else {
+                _ = check err;
```
Comment (jclark) [inline compiler/main.bal:68]: Should be just `check err;`

**Review by jclark — COMMENTED** [review]
Can you fix this last point separately?

File: compiler/main.bal (hunk 1 above)
Comment (gimantha) [reply compiler/main.bal:68]: @jclark I have a little confusion here. In spec, `checking-keyword` is only present in [call-stmt](https://ballerina.io/spec/lang/master/#call-stmt) under `statement`. So can this (`check err;`) be syntactically valid?

Comment (jclark) [reply compiler/main.bal:68]: Oh yes. You're right.  Can you create a spec issue to allow `check variable-name;` as a stmt?

Comment (gimantha) [reply compiler/main.bal:68]: sure

Comment (gimantha) [reply compiler/main.bal:68]: Can we use https://github.com/ballerina-platform/ballerina-spec/issues/588 this to track it?

Comment (jclark) [reply compiler/main.bal:68]: Yes. That's perfect.

**Pull request merged**